avocado/utils/process.py

```diff
@@ -35,9 +35,6 @@
 from .wait import wait_for
 
 LOG = logging.getLogger('avocado.test')
```

Comment:
IMO, the default namespace should be `__name__`.

Reply:
Yes, I wanted to do that, but I got a lot of problems with functional output tests, which I haven't fixed yet, so I decided to do this change in the separated PR.

Reply:
I find strange to have this separated specially because of the docstrings above. We are changing the API, so IMO we could do this with the less impact as possible to the users. Adding an argument now and then changing the default value latter, sounds strange to me.